File: Semester4/Web Programming/lab8/Student.php
<?php
class Student implements JsonSerializable {
    private $id;
    private $name;
    private $username;
    private $password;
    private $groupID;

    public function __construct($id, $name, $groupID, $username, $password) {
        $this->id = $id;
        $this->name = $name;
        $this->password = $password;
        $this->username = $username;
        $this->groupID = $groupID;
    }

    public function getId() {
        return $this->id;
    }
    public function getName() {
        return $this->name;
    }
    public function getPassword() {
        return $this->password;
    }
    public function getGroupID() {
        return $this->groupID;
    }

    public function getUsername(){
        return $this->username;
    }

    public function jsonSerialize() {
        $vars = get_object_vars($this);
        return $vars;
    }
}

?>

File: Semester4/Web Programming/lab10/WebLab10/Models/Student.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebLab10.Models
{
    public class Student
    {
        public int UserID { get; set; }
        public string Name { get; set; }
        public string Username{ get; set; }
        public string Password { get; set; }
        public string GroupID{ get; set; }
    }
}